UEBPrep/.gitignore

```diff
@@ -22,3 +22,6 @@
 
 # Vim .swp files.
 *.swp
+
+/User images
```

Comment:
Don't want to clutter up github with all the images we are generating and storing for our app.